tsconfig.json

```diff
@@ -10,6 +10,7 @@
 		"esModuleInterop": true,
 		"module": "esnext",
 		"moduleResolution": "node",
+		"noImplicitAny": false,
```

Comment:
Similar to above, except implicit any is even more dangerous in TS. `any` has a way of polluting anything it comes into contact with and basically shutting off typescript. Where there issues with implicit any that caused you to turn this off?